### DISCUSSION TITLE
Simulating Channel Flow with a Time-Varying Pressure Gradient

### DISCUSSION THREAD

**Discussion by JhonCordova** [AMReX-Combustion/PeleC#798] [Q&A] [answered]
Hello, I want to simulate a channel flow with a time-varying pressure gradient. Is this possible? Is there an option or function that I can reuse?

**Comment by baperry2** [comment, marked as answer]
The pressure-gradient-like external force specified using `pelec.external_forcing` in the input file of the ChannelFlow case is spatially and temporally constant, and that capability can only specify constant forces. You could set a temporally varying forcing through the user defined function `problem_modify_ext_sources`. This process is elaborated further in answers to #789.
